src/keyfile.rs

```diff
@@ -35,10 +35,9 @@ pub struct Crypto {
     pub cipherparams: Aes128Ctr,
     /// Cipher bytes
 	pub ciphertext: Bytes,
```

Comment:
Do you mind reformatting the whole file? Just noticed that there are some issues